dockerfiles/Dockerfile-mina-daemon

```diff
@@ -69,6 +69,7 @@ RUN echo "Building image with version $deb_version from repo $deb_release $deb_c
   && echo "deb [trusted=yes] http://packages.o1test.net $deb_codename $deb_release" > /etc/apt/sources.list.d/o1.list \
   && apt-get update --quiet --yes \
   && apt-get install --quiet --yes --allow-downgrades "${MINA_DEB}=$deb_version" \
+  && apt-get install --quiet --yes --allow-downgrade "mina-create-legacy-ledger=1.4.0beta2-georgeee-port-runtime-genesis-ledger-to-compatible-9a494bf" \
```

Comment:
I don't know if this is a good idea to depend on personal branch debian

Reply:
still waiting on https://buildkite.com/o-1-labs-2/mina-end-to-end-nightlies/builds/1750 to be able to update it

the other problem is that we need to ensure that there is a copy of this package available for every channel, otherwise the container will fail to build.

the alternative seems like a weird type of buildkite job that would have to depend on two branches simultaneously.

Reply:
@emberian you can use version: `1.4.0beta2-compatible-97f7d8c`